### PR TITLE
chore(weave): Much cleaner Object Viewer and Call View

### PR DESCRIPTION
**Pull request by tssweeney**
Better Call Page styling

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/CallStatusCodeChip.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {Chip} from '@mui/material';
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/CallStatusCodeChip.tsx:1]: This has some overlap with the work I did this week in https://github.com/wandb/weave/pull/1072

Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/CallStatusCodeChip.tsx:1]: 👍 Nice callout - let's use yours with the nice symbol!

File: weave-js/src/components/PagePanelComponents/Home/Browse3.tsx [hunk 2]
```diff
@@ -351,7 +351,12 @@ const MainPeekingLayout: FC = () => {
 
       <Box
         sx={{
-          borderLeft: '1px solid rgba(0, 0, 0, 0.12)',
+          borderLeft:
+            flexDirection === 'row' ? '1px solid rgba(0, 0, 0, 0.12)' : 'none',
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3.tsx:355]: Just noting that we should start working towards using named colors from the official palette.

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage.tsx [hunk 3]
```diff
@@ -722,3 +696,137 @@ const useCallFlattenedTraceTree = (call: WFCall) => {
     return {rows, expandKeys};
   }, [call]);
 };
+
+const CallDetails: FC<{
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage.tsx:700]: I'd like to recommend we start putting components with more than a few lines into their own files, I think it helps with readability, searching within files, etc.

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/CallStatusCodeChip.tsx (hunk 1 above)
Comment (jamie-rasmussen) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/CallStatusCodeChip.tsx:1]: OK, why don't we land this, and then I'll update that PR and re-request review.

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/KeyValueTable.tsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+  paddingLeft: '8px',
+  verticalAlign: 'top',
+  borderBottom: '1px solid rgba(224, 224, 224, 1)',
+};
+
+const KeyValueRow: React.FC<{
+  rowKey: string;
+  rowValue: any;
+}> = props => {
+  const depth = React.useContext(DepthContext);
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/KeyValueTable.tsx:91]: I'd get rid of `React.` and just add to import list as you already have for `useState`

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx [hunk 5]
```diff
@@ -218,30 +341,58 @@ export const ScrollableTabContent: React.FC<{
   );
 };
 
+const isPrimitive = (val: any) => {
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx:344]: This is defined multiple times, we should probably pull it out into a utility file

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx (hunk 5 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx:344]: Done

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/KeyValueTable.tsx (hunk 4 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/KeyValueTable.tsx:91]: Done

File: weave-js/src/components/PagePanelComponents/Home/Browse3.tsx (hunk 2 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3.tsx:355]: I need to do this moving forward - you are correct. I fixed this case, but probably need to write a script to look through all of our weave js files and convert to the closest common color

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage.tsx (hunk 3 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage.tsx]: Done

**Pull request merged**